Examples/BareBones/BareBones/AppStatsViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  AppStatsViewController.swift
+//  BareBones
+//
+//  Created by Edward Smith on 10/3/17.
+//  Copyright © 2017 Branch. All rights reserved.
+//
+
+import UIKit
+import Branch
+
+class AppStatsViewController: UIViewController {
+
+    @IBOutlet weak var statsLabel: UILabel!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        NotificationCenter.default.addObserver(
```

Comment:
You might want to call `NotificationCenter.default.removeObserver(self)` in deinit or viewWillUnload or something, as you did in AppStats. This is just a matter of best practices, since in practice the root VC never goes away, but probably a good idea in an example app, especially when adding a new feature.